insights/parsers/auditctl_status.py

```diff
@@ -60,3 +67,43 @@ def parse_content(self, content):
                         self.data[k.strip()] = int(v.strip())
                     except ValueError:
                         continue
+
+
+@parser(Specs.auditctl_rules)
+class AuditRules(CommandParser, list):
+    """
+    Class for parsing the `auditctl -l` command.
+    All lines are stored in a list.
+
+    Typical output of the command is::
+
+        -w /etc/selinux -p wa -k MAC-policy
+        -a always,exit -F arch=b32 -S chmod -F exit=-EACCES -F auid>=1000 -F auid!=-1 -F key=access
+        -a always,exit -F arch=b64 -S chmod -F exit=-EACCES -F auid>=1000 -F auid!=-1 -F key=access
+        -a always,exit -F arch=b32 -S chmod -F exit=-EPERM -F auid>=1000 -F auid!=-1 -F key=access
+        -a always,exit -F arch=b64 -S chmod -F exit=-EPERM -F auid>=1000 -F auid!=-1 -F key=access
+        -a always,exit -F arch=b32 -S chown -F exit=-EACCES -F auid>=1000 -F auid!=-1 -F key=access
+        -a always,exit -F arch=b64 -S chown -F exit=-EACCES -F auid>=1000 -F auid!=-1 -F key=access
+        -a always,exit -F arch=b32 -S chown -F exit=-EPERM -F auid>=1000 -F auid!=-1 -F key=access
+        -a always,exit -F arch=b64 -S chown -F exit=-EPERM -F auid>=1000 -F auid!=-1 -F key=access
+
+    Examples:
+        >>> type(audit_rules)
+        <class 'insights.parsers.auditctl_status.AuditRules'>
+        >>> len(audit_rules)
+        9
+        >>> '-w /etc/selinux -p wa -k MAC-policy' in audit_rules
+        True
+
+    Raises:
+        SkipException: When there are not rules.
+    """
+    def __init__(self, context):
+        super(AuditRules, self).__init__(context, extra_bad_lines=['no rules'])
```

Comment:
This looks like a warning message from the command when there are no rules rather than the bad lines. I think we can use this to raise an exception when no rules are found.

Reply:
@psachin, updated as suggested.